src/shared/platform/lind_platform.c

```diff
@@ -5,26 +5,33 @@
  *      Author: sji
  */
 
+#undef _POSIX_C_SOURCE
+
 #include <Python.h>
 #include <errno.h>
 
-#include "native_client/src/shared/platform/nacl_log.h"
 #include "native_client/src/shared/platform/lind_platform.h"
+#include "native_client/src/shared/platform/nacl_log.h"
+#include "native_client/src/trusted/service_runtime/include/bits/nacl_syscalls.h"
 
-PyObject* repylib = NULL;
-PyObject* code = NULL;
-PyObject* context = NULL;
+int lind_syscall_counter;
+int lind_syscall_invoked_times[NACL_MAX_SYSCALLS];
+double lind_syscall_execution_time[NACL_MAX_SYSCALLS];
 
-static int initialized = 0;
+PyObject *py_repylib;
+PyObject *py_code;
+PyObject *py_context;
+
+static int initialized;
 
 #define REPY_RELPATH "../repy/"
```

Comment:
Does this path definition require corresponding file directory to be setup during installation? If so, perhaps we should provide comments and documentation to make sure things are setup correctly.

Reply:
Yeah, the macro just obfuscates this part, as it's really only used in one place. Just going to remove it and use a string literal.